docs/source/models.rst

```diff
@@ -360,17 +366,23 @@ platform for real-time serving.
 
     sample_input='
     {
-         "residual sugar": {"0": 20.7},
-         "alcohol": {"0": 8.8},
-         "chlorides": {"0": 0.045},
-         "density": {"0": 1.001},
-         "sulphates": {"0": 0.45},
-         "total sulfur dioxide": {"0": 170.0},
-         "fixed acidity": {"0": 7.0},
-         "citric acid": {"0": 0.36},
-         "pH": {"0": 3.0},
-         "volatile acidity": {"0": 0.27},
-         "free sulfur dioxide": {"0": 45.0}
+        "columns": [
+            "alcohol",
+            "chlorides",
+            "citric acid",
+            "density",
+            "fixed acidity",
+            "free sulfur dioxide",
+            "pH",
+            "residual sugar",
+            "sulphates",
+            "total sulfur dioxide",
+            "volatile acidity"
+        ],
+        "index": [0],
```

Comment:
Could we document what index is in our RST docs? Since users may want to construct this by hand. That and/or link to the split documentation in Pandas.

Reply:
It turns out the `index` is an optional field used to provide labels for each row of the dataframe. I've removed it from the example and explained that it's an optional field. I also added an explanation for the format of `sample_input` that refers to the MLflow AzureML documentation.